example/README.md

```diff
@@ -0,0 +1,39 @@
+# React Native Example App
+
+## Running
+
+```sh
+yarn install
+cd example
+yarn install
+```
+
+### iOS
+
+To run for iOS, you need the following pre-requisites:
+
+* cocoapods (install from Homebrew)
+* Xcode
+
+Generate the Pod project with:
+
+```sh
+cd ios
+pod install
+```
+
+Then open in Xcode:
```

Comment:
Did running from command line with `yarn ios` not work?

Reply:
Oh I guess entitlements may be an issue.

Reply:
yarn ios didn't work for me. also, since I'm on a M1 Mac, iphone simulator didn't work.